types/react-payment-inputs/images/index.d.ts

```diff
@@ -0,0 +1,17 @@
+import React = require("react");
+
+interface CardImages {
```

Comment:
```suggestion
export interface CardImages {
```
(just export to avoid `export {}`, just simply do not re-export from `index.d.ts`